### PR TITLE
edit-filestruc-datatypes

### PR DESCRIPTION
**Pull request by krishahn**
minor wordsmithing

### CODE REVIEW

File: versions/3.0.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ The OAS representation of the API is made of a single file.
 However, parts of the definitions can be split into separate files, at the discretion of the user.
 This is applicable for `$ref` fields in the specification as follows from the [JSON Schema](http://json-schema.org) definitions.
 
-By convention, it is RECOMMENDED that the OpenAPI Specification (OAS) file be named `openapi.json` or `openapi.yaml`.
+It is RECOMMENDED that the OpenAPI Specification (OAS) file be named following convention: `openapi.json` or `openapi.yaml`.
```
Comment (DavidBiesack) [inline versions/3.0.md]: reads a little stiff to me. Suggest "named using convention `openapi.json` ... (without the colon).

Comment (DavidBiesack) [inline versions/3.0.md]: Also, **this** page is the OpenAPI Specification. The openapi.json file is **not** the specification, it is the **OpenAPI definition**.

I suggest we define that as a term - "an **OpenAPI definition** is a document that describes an OpenAPI using the OAS".

Comment (RobDolinMS) [inline versions/3.0.md]: Per @darrelmiller this should be an Open API Definition File (OADF)

Comment (darrelmiller) [inline versions/3.0.md]: Use the phrase "OpenAPI definition file" with no acronym.

**Comment by RobDolinMS** [comment]
#TDC: Thumbs-up

**Pull request merged**